gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTableFilter.java

```diff
@@ -32,7 +32,15 @@ public KnoxShellTableFilter table(KnoxShellTable table) {
   }
 
   public KnoxShellTableFilter name(String name) {
-    index = tableToFilter == null ? -1 : tableToFilter.headers.indexOf(name);
+    for (int i = 0; i < tableToFilter.headers.size(); i++) {
+      if (tableToFilter.headers.get(i).equalsIgnoreCase(name)) {
```

Comment:
Is it possible to store the headers in a case insensitive way? ie: lowercase on insert? This would avoid having to deal with case insensitive headers other places.